clippy_lints/src/cargo/mod.rs

```diff
@@ -161,6 +161,15 @@ declare_clippy_lint! {
     /// [dependencies]
     /// regex = "*"
     /// ```
+    /// Use instead:
+    /// ```toml
+    /// [dependencies]
+    /// # allow patch updates, but not minor or major version changes
+    /// some_crate_1 = "~1.2.3"
+    ///
+    /// # pin the version to a specific version
+    /// some_crate_2 = "1.2.3"
```

Comment:
Actually `some_crate_2 = "1.2.3"` is effectively equal to `some_crate_2 = "^1.2.3"`. If you want to pin, you need `some_crate_2 = "=1.2.3"` instead.